src/components/UserAuth/Login.jsx

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+import "../../styles/root.css";
+import { steve } from "../../assets/images";
+import { Link } from "react-router-dom";
+
+export function Login() {
+	return (
+		<>
+			<div
```

Comment:
Here you can remove these fragments as you are already enclosing your code inside a div